Services/BrokerService/Utils/BrokerFactory.cs

```diff
@@ -1,13 +1,22 @@
-﻿using System;
+﻿using BrokerService;
+using DTO;
 using Interfaces;
+using Kernel.CustomExceptions;
+using TinkoffIntegrationLib;
 
 namespace BrokerServices.Utils
 {
-    public static class BrokerFactory
+    static class BrokerFactory
     {
-        public static IBroker Create(BankType bankType)
+        internal static IBroker Create(BankType bankType)
         {
-            throw new NotImplementedException();
+            var brokerData = new CreateBrokerData
```

Comment:
If it is constants why don't you write default values in constructor?

Reply:
> If it is constants why don't you write default values in constructor?

Wrote the default value for properties in the CreateBrokerData class.